stylegan_code_finder/lightning_modules/trans_u_net_lightning.py

```diff
@@ -12,9 +12,10 @@
 class TransUNetSegmenter(BaseSegmenter):
     def __init__(self, configs: dict):
         super().__init__(configs)
-        self.ce_loss = nn.CrossEntropyLoss()
-        self.dice_loss = DiceLoss(configs['num_classes'])
+        #self.ce_loss = nn.CrossEntropyLoss()

```

Comment:
All the commented lines have to be removed or used in the end. If you want to keep both versions, make it customizable via parameters